Sources/PrimerSDK/Classes/Data Models/PrimerPaymentMethodType.swift

```diff
@@ -10,6 +10,7 @@ internal enum PrimerPaymentMethodType: String, Codable, CaseIterable, Equatable,
     
     case adyenAlipay        = "ADYEN_ALIPAY"
     case adyenBlik          = "ADYEN_BLIK"
+    case adyenBancontact    = "ADYEN_BANCONTACT_CARD"
```

Comment:
Can you check whether there's a chance to have `adyenBancontact` in the future? If that's the case, then it would be better to rename `adyenBancontact` to `adyenBancontactCard`

Reply:
Good one 👍 - updated as I've seen there were discussions about this naming. Better to have an explicit one, matching the classes' names we have for the CardData